lib/opentypesvg/fonts2svg.py

```diff
@@ -260,6 +280,12 @@ def get_options(args):
         dest='glyphsets_union',
         help="do union (instead of intersection) of the fonts' glyph sets."
     )
+    parser.add_argument(
+        '-av', '--adjust-viewbox',
+        action='store_true',
+        dest='adjust_view_box_to_glyph',
+        help="adjust the viewbox to the glyph (descender calculated and lineGap in height)."
```

Comment:
This description is confusing. What about "vertically center the viewBox on the bounding box of all glyphs" ?